samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -1081,6 +1114,7 @@ Device Firmware Update (DFU)
 
 This sample uses following components for the DFU functionality:
 
-* :ref:`MCUboot bootloader <mcuboot:mcuboot_ncs>`
+* :ref:`MCUboot bootloader <mcuboot:mcuboot_ncs>` (for MCUboot-based build targets)
+* :ref:`SUIT <ug_nrf54h20_suit_dfu>` (for SUIT-based build targets)
```

Comment:
build targets or board targets? (this is always difficult to me)

Reply:
should be board targets, I will fix that

Reply:
fixed